vars/main.yml

```diff
@@ -339,6 +341,20 @@ pgbackrest_conf:
     - {option: "process-max", value: "2"}
     - {option: "recovery-option", value: "recovery_target_action=promote"}
 #    - {option: "", value: ""}
+
+pgbackrest_srv_conf:
+  global:  # [global] section
+    - {option: "log-level-file", value: "detail"}
+    - {option: "log-path", value: "/var/log/pgbackrest"}
+    - {option: "repo1-type", value: "{{ pgbackrest_repo_type |lower }}"}
+#   - {option: "repo1-retention-full", value: 5} #  Take care, not to fill up your disks with backups
+    - {option: "repo1-path", value: "/var/lib/pgbackrest"}
+    - {option: "repo1-host-user", value: "{{ pgbackrest_repo_user }}"}
+  stanza:
+    - {option: "process-max", value: "2"}
+    - {option: "recovery-option", value: "recovery_target_action=promote"}
+    - {option: "backup-standby", value: "y"}  # Don't take backups from a primary, move the work to a secondary, especially because of compression
```

Comment:
I suggest replacing the comment with the one specified in the pgbackrest documentation

`# Enable backup from standby to reduce load on the primary cluster. `